bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -140,6 +264,7 @@ private void disposeGardena() {
         final GardenaSmart gardenaSmart = this.gardenaSmart;
         if (gardenaSmart != null) {
             gardenaSmart.dispose();
+            this.gardenaSmart = null;
```

Comment:
Move below the `if` scope for consistency with pattern in line 247-250?

Reply:
Done.